Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -275,13 +283,53 @@ def __init__(self):
             callback=self._on_target_class_changed,
             contentsLength=8)
 
+        self.modcompbox = box = gui.vBox(self.controlArea, "Model Comparison")
+        gui.comboBox(
+            box, self, "comparison_criterion", model=PyListModel(),
+            callback=self.update_comparison_table)
+
+        hbox = gui.hBox(box)
+        gui.checkBox(hbox, self, "use_rope",
+                     "Negligible difference: ",
+                     callback=self.update_comparison_table)
+        gui.lineEdit(hbox, self, "rope", validator=QDoubleValidator(),
```

Comment:
Why is this not a spinbox? 
It should probably disabled when `use_rope` is not checked.

Reply:
It's not a spin box because it has no defined range. It can refer to AUC, that is, between 0 and 1, or it can be RMSE, which is between 0 and infinity -- it can easily be 100000.

Disabling it would make sense, I'll do that.

Reply:
I added disabling but didn't like it. Let's let the user change the line edit first and then enable it, if (s)he wishes.

I added a method `_on_use_rope_changed`. You can add a line `self.controls.rope.setEnabled(self.use_rope)` and see for yourself that you won't like it. :)

Reply:
This is a won't fix. :)